oauth2_http/java/com/google/auth/oauth2/OAuth2Credentials.java

```diff
@@ -154,7 +156,9 @@ public void refresh() throws IOException {
     synchronized (lock) {
       requestMetadata = null;
       temporaryAccess = null;
-      useAccessToken(Preconditions.checkNotNull(refreshAccessToken(), "new access token"));
+      useAccessToken(
+          Preconditions.checkNotNull(refreshAccessToken(), "new access token"),
```

Comment:
This is a very strange and possibly inappropriate use of Preconditions.

Reply:
I agree, it's been there for years.

Filed https://github.com/googleapis/google-auth-library-java/issues/510 to track.